spec/CLI.spec.js

```diff
@@ -294,4 +294,20 @@ describe('execution', () => {
       done.fail(data.toString());
     });
   });
+
+  it('can start Parse Server with auth via CLI', done => {
+    const env = { ...process.env };
+    env.NODE_OPTIONS = '--dns-result-order=ipv4first';
```

Comment:
There seems to be an issue with Node 12 compatibility:

> 1) execution can start Parse Server with auth via CLI
>   - Failed: node: --dns-result-order= is not allowed in NODE_OPTIONS

https://github.com/parse-community/parse-server/actions/runs/5386228844/jobs/9776051408?pr=8669#step:9:3989

It seems this needs a condition: Node >= 14

Reply:
@mtrezza I quickly patched the 5.x.x CLI.spec it and tried to test with node 12 however node 12 is no longer support, see package.json:

```
  "engines": {
    "node": ">=14.21.0 <17 || >=18 <19"
  },

``` 

Assuming this is correct (which seems to make sense), the fix should be to remove testing on node 12?

Reply:
I resolved this because the LTS branch isn't tested on node 18, which requires the dns order

Reply:
@FransGH Node 12 is still supported by Parse Server 5. You must have looked at a different branch.